Dockerfile.template

```diff
@@ -1,8 +1,8 @@
-FROM balenalib/%%BALENA_ARCH%%-debian:latest-build
+FROM debian:bookworm-slim
 
 WORKDIR /hello-world
 
-RUN install_packages gcc cmake libboost-context-dev libboost-dev curl libpq-dev cmake 
+RUN apt-get update && apt-get install -y gcc cmake libboost-context-dev libboost-dev curl libpq-dev cmake 
```

Comment:
Still missing `g++`, so it might be simpler to just install [build-essential](https://packages.debian.org/bookworm/build-essential) and include most build dependencies.